src/openrct2/interface/Viewport.cpp

```diff
@@ -45,7 +45,7 @@
 
 using namespace OpenRCT2;
 
-//#define DEBUG_SHOW_DIRTY_BOX
+// #define DEBUG_SHOW_DIRTY_BOX
```

Comment:
I guess this is to satisfy clang-format? If so, strange that it only comes up now.

Reply:
Yes, it's because of clang-format. I've seen this before, don't know why they don't show up earlier.

Reply:
Let’s get rid of it altogether (in a separate commit), we shouldn’t have commented-out code anyway.

Reply:
Good point, I deleted it. I also fixed a typo which I only just noticed in the global field.